### DISCUSSION TITLE
Can trace 's timestamp precision reach microsecond level？

### DISCUSSION THREAD

**Discussion by bigbanglwb** [apache/skywalking#9522] [Q&A]
I tried to research the overhead of communication in microservice with Skywalking. However, the timestamp precision is millisecond level as the figure show and not able to analyze the overhead. I want to know if there is some configuration can make Skywalking  trace's timestamp precision reach microsecond level?
![image](https://user-images.githubusercontent.com/43382319/186602724-ffd8aa8e-da2b-4f6b-8996-f2c96300f1d0.png)

I know jaeger can achieve microsecond level. It gives millisecond level timestamp of start time and microsecond level duration.

**Comment by wu-sheng** [comment]
No, there isn't. Because we don't think it is reasonable to use an APM tool to diagnose a perf issue less than `1 ms`. 
There is no case for that. Creating a span would take nearly `0.01 - 0.1` ms even ignoring the GC and serialization(in report) cost.
This is never about what unit we provide.

**Reply by bigbanglwb** [reply]
thank you
